data/stereoseq_mouse_embryo/stereoseq_mouse_embryo.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+import urllib.request
+from urllib.parse import urlparse
+import os
+import anndata
+import argparse
+import shutil
+import pandas as pd
+import scipy
+import json
+import tempfile
+
+# 6 available but only 2 contain region label and coordinates
+
+sample_info=pd.read_csv('sample_info.csv')
+
+LINKS = sample_info["download"].tolist()
+
```

Comment:
Hi, thanks for implementing this dataset. Including an additional file in the folder and sourcing it can be problematic when running the pipeline in snakemake. Can you incorporate the `sample_info.csv` into the main script and delete that file?

Since only `download` and `sample_name` are used. something like: 
```
sample_name = ['E9.5_E1S1', 'E9.5_E2S1', 'E9.5_E2S2', 'E9.5_E2S3', 'E9.5_E2S4',
       'E10.5_E1S1', 'E10.5_E1S2', 'E10.5_E1S3', 'E10.5_E2S1',
       'E11.5_E1S1', 'E11.5_E1S2', 'E11.5_E1S3', 'E11.5_E1S4',
       'E12.5_E1S1', 'E12.5_E1S2', 'E12.5_E1S3', 'E12.5_E1S4',
       'E12.5_E1S5', 'E12.5_E2S1', 'E13.5_E1S1', 'E13.5_E1S2',
       'E13.5_E1S3', 'E13.5_E1S4', 'E14.5_E1S1', 'E14.5_E1S2',
       'E14.5_E1S3', 'E14.5_E1S4', 'E14.5_E1S5', 'E14.5_E2S1',
       'E14.5_E2S2', 'E15.5_E1S1', 'E15.5_E1S2', 'E15.5_E1S3',
       'E15.5_E1S4', 'E15.5_E2S1', 'E16.5_E1S1', 'E16.5_E1S2',
       'E16.5_E1S3', 'E16.5_E1S4', 'E16.5_E1S5', 'E16.5_E2S10',
       'E16.5_E2S11', 'E16.5_E2S12', 'E16.5_E2S13', 'E16.5_E2S1',
       'E16.5_E2S2', 'E16.5_E2S3', 'E16.5_E2S4', 'E16.5_E2S5',
       'E16.5_E2S6', 'E16.5_E2S7', 'E16.5_E2S8', 'E16.5_E2S9']
download_link = [f"https://ftp.cngb.org/pub/SciRAID/stomics/STDS0000058/stomics/{sample}.MOSTA.h5ad" for sample in sample_name]
```
could work the same as having a csv attached.